### ISSUE TITLE
`require()` Caching - causes issues with reports

### ISSUE THREAD

**Issue by jxmot** [jxmot/router-log-monitor#17]
The fact that `require()` will cache modules, and will not reload them even with explicit `require()` calls.

A work-around or means to disable the caching is needed.

**Comment by jxmot** [comment]
This is the work-around (tested OK) - 

```
    // NOTE: require() does caching of modules, info:
    //      https://bambielli.com/til/2017-04-30-node-require-cache/
    //
    // Also using every() instead of forEach() to allow the ability
    // to break the loop:
    //      https://masteringjs.io/tutorials/fundamentals/foreach-break
    Object.keys(require.cache).every( (key) => {
        if(key.includes('reportdefs.js')) {
            // remove the module from the cache
            delete require.cache[key];
            // done!
            return false;
        }
        return true;
    });

    const reportdefs = require('./reportdefs.js');
```

**Comment by jxmot** [comment]
resolved in the 20210531-logreporter branch
